tests/integration-tests/tests/scaling/test_scaling.py

```diff
@@ -184,6 +185,33 @@ def test_hit_scaling(scheduler, region, instance, pcluster_config_reader, cluste
         num_dynamic_nodes=3,
         dynamic_instance_type=instance,
     )
+    assert_no_errors_in_logs(remote_command_executor, scheduler)
+
+
+@pytest.mark.regions(["us-west-1"])
+@pytest.mark.instances(["c5.xlarge"])
+@pytest.mark.schedulers(["slurm"])
+@pytest.mark.oss(["alinux2"])
+@pytest.mark.usefixtures("region", "os")
+@pytest.mark.hit_rare_failure
+def test_hit_rare_failure(scheduler, region, instance, pcluster_config_reader, clusters_factory, test_datadir):
```

Comment:
shall we move these 2 slurm tests (test_hit_scaling, test_hit_rare_failure) to the slurm specific module and rename them to something like: test_slurm_scaling and test_error_handling?

Reply:
Done